utils/sub8_ros_tools/sub8_ros_tools/geometry_helpers.py

```diff
@@ -70,3 +73,36 @@ def compose_transformation(R, t):
     transformation[3, :3] = t
     transformation[3, 3] = 1.0
     return transformation
+
+
+def quat_to_rotvec(q):
+    ''' Turn a quaternion into a axis rotation vector [x,y,z] '''
+    if q[3] < 0:
+        q = -q
+    q = transformations.unit_vector(q)
+    angle = math.acos(q[3])*2
+    axis = normalize(q[0:3])
+    return axis * angle
+
+def quat_to_np(q):
+    ''' Turn a quaternion into a numpy array '''
+    array = np.zeros((4))
+    array[0] = q.x
+    array[1] = q.y
+    array[2] = q.z
+    array[3] = q.w
+    return array
+
+def np_to_quat(array):
+    ''' Turn a numpy array into a ROS quaternion '''
```

Comment:
This is a good idea - let's put it in msg_helpers instead of geometry_helpers so it can live with its friends